libs/components/layout/src/lib/modules/box/box.component.html

```diff
@@ -10,52 +10,56 @@
 >
   <div class="sky-box-header-content">
     @if (headingText) {
-      @switch (headingLevel) {
-        @case (3) {
-          <h3
-            [id]="headerId"
-            [class]="headingClass + ' sky-margin-inline-xs'"
-            [class.sky-screen-reader-only]="headingHidden"
-          >
-            {{ headingText }}
-          </h3>
+      <div class="sky-box-header-right">
+        @switch (headingLevel) {
+          @case (3) {
+            <!-- display: inline -->
```

Comment:
Are these comments still needed?

Reply:
yes--they prevent prettier from formatting the heading elements in the way that adds extra space before and after. It is an alternative to using `skyTrim`